be/src/olap/tablet_schema_cache.h

```diff
@@ -52,6 +55,25 @@ class TabletSchemaCache {
         return iter->second;
     }
 
+private:
+    /**
+     * @brief recycle when TabletSchemaSPtr use_count equals 1.
+     */
+    void _recycle() {
+        for (;;) {
+            std::this_thread::sleep_for(
+                    std::chrono::seconds(config::tablet_schema_cache_recycle_interval));
```

Comment:
Could we remove this config? There are too many configs in doris? Do we really need it as a config?